include-build/roborazzi-core/src/commonJvmMain/kotlin/com/github/takahirom/roborazzi/CaptureResults.kt

```diff
@@ -1,24 +1,21 @@
 package com.github.takahirom.roborazzi
 
 import com.google.gson.Gson
-import com.google.gson.JsonArray
 import com.google.gson.JsonObject
+import com.google.gson.JsonParser
 import java.io.File
 import java.io.FileReader
 
 data class CaptureResults(
-  val summary: ResultSummary,
+  val resultSummary: ResultSummary,
   val captureResults: List<CaptureResult>
 ) {
   fun toJson(): JsonObject {
-    val json = JsonObject()
-    json.add("summary", summary.toJson())
-    val resultsArray = JsonArray()
-    captureResults.forEach { result ->
-      resultsArray.add(result.toJson())
+    val captureResultsObject = object {
+      val summary = resultSummary
+      val results = captureResults.map { it.toJson() }
```

Comment:
Because a new object cannot be created if the variable named `summary` has a duplicate name, the name of the other one was modified.